### DISCUSSION TITLE
Why does the UI not format output from GPT?

### DISCUSSION THREAD

**Discussion by javapro13** [Azure-Samples/chat-with-your-data-solution-accelerator#516] [Q&A]
We are using. this and for the same model on the backend, the UI is not formatting any output. Check out the attached screen capture 1. WE can create a OpenAI resource and deploy it as web app and the output is formatted so much better. Why is that?

Currently we are only able to use the backend to upload docs and using an OpenAI deploy as web app for the frontend? Are we missing something?
<img width="993" alt="Screenshot 2024-03-21 at 4 24 24 PM" src="https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/10634366/f771a218-e7a2-4dfc-867a-0f7df35c1a71">
<img width="1066" alt="Screenshot 2024-03-21 at 4 25 42 PM" src="https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/10634366/db5aaf01-8f73-490b-bee2-82f89b5dbcc7">

**Comment by adamdougal** [comment]
Heya @javapro13! 

There are a few reasons why this might be happening, but my first thought is potential differences between the system prompt. Have you tried changing that for this project?

We are currently in the process of changing the default system prompt for this project, once we've done that I'll let you know so you can try again.

**Comment by ytausch** [comment]
I have the same issue when just deploying the project without modifying anything.
